features/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/webapp/pages/asset.jag

```diff
@@ -213,11 +214,15 @@ require('/modules/store.js').exec(function (ctx) {
             assetsFromProvider['overview_provider'] = provider;
             assetsFromProvider['type'] = type;
             assetsFromProvider['assets'] = store.search(searchOptions, paging)[asset.type] || [];
-            businessOwner = storeHostObj.getBusinessOwner(asset.id);
-            if(businessOwner != null && businessOwner.businessOwnerName != null) {
-                businessOwnerCustomData = require('../themes/store/js/logic/businessOwners/businessOwners.js').transform(businessOwner);
-                ownerName = businessOwnerCustomData[0].value;
+            if(asset.attributes.overview_businessOwner != "null") {
+                businessOwner = storeHostObj.getBusinessOwner(asset.attributes.overview_businessOwner);
+                if(businessOwner != null && businessOwner.businessOwnerName != null) {
+                    businessOwnerCustomData = require('../themes/store/js/logic/businessOwners/businessOwners.js').transform(businessOwner);
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5